doc-content/drools-docs/src/main/asciidoc/DecisionEngine/cep-con.adoc

```diff
@@ -0,0 +1,40 @@
+[id='cep-con_{context}']
+
+= Complex event processing (CEP)
+
+In {PRODUCT}, an event is a record of a significant change of state in the application domain. Depending on how the domain is modeled, the change of state may be represented by a single event, multiple atomic events, or hierarchies of correlated events. From a complex event processing (CEP) perspective, an event is a type of fact or object and a business rule is a definition of how to react to the data from that fact or object. For example, in a stock broker application, a change in security prices, a change in ownership from seller to buyer, or a change in an account holder's balance are all considered to be events because a change has occurred in the state of the application domain.
```

Comment:
I'd underline the notion of type here. An event is not just a fact but a fact that happens in a precise point in time.

Reply:
Sure. How about this. The bolded portions are what I've added. Is that better or overkill?

> In {PRODUCT}, an event is a record of a significant change of state in the application domain **at a point in time**. Depending on how the domain is modeled, the change of state may be represented by a single event, multiple atomic events, or hierarchies of correlated events. From a complex event processing (CEP) perspective, an event is a type of fact or object t**hat occurs at a specific point in time**, and a business rule is a definition of how to react to the data from that fact or object. For example, in a stock broker application, a change in security prices, a change in ownership from seller to buyer, or a change in an account holder's balance are all considered to be events because a change has occurred in the state of the application domain **at a given time**.